js/maps-setup.js

```diff
@@ -86,7 +152,7 @@ function initializeMap() {
             window_content: all_my_markers[j].window_content});
 
         // this next line is ugly, and you should change it to be prettier.
-        // be careful not to introduce syntax errors though.  
+        // be careful not to introduce syntax errors though.
       legendHTML +=
         "<div class=\"pointer\" onclick=\"locateMarker(my_markers[" + j + "])\"> " +
```

Comment:
With so many markers, your legend is getting too tall to navigate. Maybe put just `marker.tytle` instead of  `marker.window_content`, to compress.